opentech/apply/projects/views.py

```diff
@@ -169,6 +231,38 @@ def form_valid(self, form):
         return response
 
 
+class DeletePaymentRequestView(DeleteView):
+    model = PaymentRequest
+    pk_url_kwarg = 'payment_request_id'
+
+    def dispatch(self, request, *args, **kwargs):
+        self.project = get_object_or_404(Project, pk=self.kwargs['pk'])
+
+        is_admin = request.user.is_apply_staff
+        is_owner = request.user == self.project.user
+        if not (is_owner or is_admin):
```

Comment:
This doesn't match with the logic in the templatetag - only owners can delete

Reply:
Pulled the templatetag logic into the model and called that here and in the templatetag so there's one source of truth for that logic.  Updated tests to reflect that too.